frontend/src/components/common/Verortung.vue

```diff
@@ -188,16 +312,29 @@ export default class Verortung extends Mixins(GeodataEaiApiRequestMixin, SaveLea
     );
   }
 
+  get kitaplanungsbereiche(): Array<KitaplanungsbereichDto> {
+    return _.sortBy(_.toArray(this.verortungModel?.kitaplanungsbereiche), ["nummer"]);
```

Comment:
Das mit `nummer` wird hier nicht funktionieren, da es das Attribute in Kitaplanungsbereich nicht gibt. Den Fehler hatte ich in einem vorherigen Commit eigentlich schon behoben gehabt

```suggestion
    return _.sortBy(_.toArray(this.verortungModel?.kitaplanungsbereiche), (k) => Number(k.kitaPlbT));
```

Reply:
Das Attribut gibt nun. Es wurde in der GeodatenEAI nicht gemappt. Nun wird es mitgeliefert.